doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,37 +57,45 @@ Values are provided in kilobytes (KB).
+      | :ref:`Lock <matter_lock_sample>` (Debug)                           |            28 |               770 |              4 |         32 |         834 |                             177 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Lock <matter_lock_sample>` (Lto debug)                       |            28 |               705 |              4 |         32 |         769 |                             181 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Lock <matter_lock_sample>` (Release)                         |            28 |               654 |              4 |         32 |         718 |                             172 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Debug)                   |            28 |               710 |              4 |         32 |         774 |                             174 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Lto debug)               |            28 |               653 |              4 |         32 |         717 |                             178 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Release)                 |            28 |               615 |              4 |         32 |         679 |                             168 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)               |            28 |               761 |              4 |         32 |         825 |                             175 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Thermostat <matter_thermostat_sample>` (Lto debug)           |            28 |               696 |              4 |         32 |         760 |                             179 |
```

Comment:
```suggestion
      | :ref:`Thermostat <matter_thermostat_sample>` (LTO, Debug)          |            28 |               696 |              4 |         32 |         760 |                             179 |
```

`Debug` and `Release` are established terms, and `LTO` is just an additional build feature